node/resolvers/Queries/Settings.ts

```diff
@@ -52,7 +53,38 @@ const B2BSettings = {
       clients: { sellers },
     } = ctx
 
-    return (await sellers.getSellers())?.items
+    try {
+      return (await sellers.getSellers())?.items
+    } catch (e) {
+      if (e.message) {
+        throw new GraphQLError(e.message)
+      } else if (e.response?.data?.message) {
+        throw new GraphQLError(e.response.data.message)
+      } else {
+        throw new GraphQLError(e)
+      }
+    }
+  },
```

Comment:
as we are creating a new method, I think it is better if we do not touch this part of the code (to avoid changing behaviour of the existing method)

I know this is improving the code, but still, it could cause unexpected behaviour for current use cases.